Sources/Core/GTMSessionFetcher.m

```diff
@@ -498,22 +499,22 @@ - (void)beginFetchWithDelegate:(nullable id)target
   [self beginFetchWithCompletionHandler:handler];
 }
 
-// Helper to begin a delayed state. If the fetch has already been stopped, then it will
-// trigger a needed callback instead of setting the state. It returnsYES/NO based on if
-// starting up of the fetch should continue.
-- (BOOL)startDelayState:(GTMSessionFetcherDelayState)delayState __attribute__((objc_direct)) {
+// Helper to enter a new StartingState. If the fetch has already been stopped, then it will trigger
+// a needed callback instead of setting the state. It returns YES/NO based on if starting up of the
+// fetch should continue.
+- (BOOL)startingState:(GTMSessionFetcherStartingState)newState __attribute__((objc_direct)) {
   BOOL stopped = NO;
   @synchronized(self) {
     GTMSessionMonitorSynchronized(self);
 
     GTMSESSION_ASSERT_DEBUG(
-        (_delayState == kDelayStateNotDelayed || _delayState == kDelayStateStartingUp),
-        @"Unexpected internal state: %lu", (unsigned long)_delayState);
+        (_startingState == kStartingStateNone || _startingState == kStartingStateStartingUp),
+        @"Unexpected internal state: %lu", (unsigned long)_startingState);
```

Comment:
nit: "Unexpected starting state"? Just to help give folks an idea what to look for on failure.

Reply:
Done.